packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -246,10 +385,15 @@ export default function () {
                 let defNode = t.identifier("default");
                 if (id) {
                   addTo(exports, id.name, defNode);
-                  topNodes.push(buildExportsAssignment(defNode, id));
+                  topNodes.push(spec ? specBuildExportDefault({ EXPORTS: exportsObj, VALUE: id }) : buildExportsAssignment(defNode, id));
```

Comment:
These have the potential to still be live-binding if something assigns to `id`, will we need to use `specBuildExport` for that case?